modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefinePrefiltersAndSortsToRetrieveOnlyASelectionOfEntriesInThePreferredOrder/AllowUsersToDefineSorts.testcase

```diff
@@ -0,0 +1,299 @@
+@component-name = "portal-headless"
+definition {
+
+	property custom.properties = "feature.flag.LPS-178642=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Object";
+	property testray.main.component.name = "Headless Builder";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given objects definiton Student with Integer field 'age'") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				businessType = "Integer",
+				en_US_label = "student",
+				en_US_plural_label = "students",
+				name = "Student",
+				objectDefinitionExternalReferenceCode = "student",
+				requiredStringFieldName = "age",
+				requiredStringFieldNameExternalReferenceCode = "studentAge");
+		}
+
+		task ("And Given Student entries with age '0', '25', '101', '-1' created") {
+			CustomObjectAPI.createObjectEntryWithFields(
+				en_US_plural_label = "students",
+				fieldName = "age",
+				fieldValue = 0);
+
+			CustomObjectAPI.createObjectEntryWithFields(
+				en_US_plural_label = "students",
+				fieldName = "age",
+				fieldValue = 25);
+
+			CustomObjectAPI.createObjectEntryWithFields(
+				en_US_plural_label = "students",
+				fieldName = "age",
+				fieldValue = 101);
+
+			CustomObjectAPI.createObjectEntryWithFields(
+				en_US_plural_label = "students",
+				fieldName = "age",
+				fieldValue = "-1");
+		}
+
+		task ("Given ApiApplication with an endpoint of scope Company created and published") {
```

Comment:
And Given?

Reply:
mum...I copy from the ticket, I will update this